include/drm/drm_plane.h

```diff
@@ -179,6 +179,24 @@ struct drm_plane_state {
 	 */
 	enum drm_color_range color_range;
 
+	/**
+	 * @chroma_siting_h:
+	 *
+	 * Location of chroma samples horizontally compared to luma
+	 * 0 means chroma is sited with left luma
+	 * 0x8000 is intersitial. 0x10000 is sited with right luma
+	 */
+	int32_t chroma_siting_h;
+
+	/**
+	 * @chroma_siting_v:
+	 *
+	 * Location of chroma samples vertically compared to luma
+	 * 0 means chroma is sited with top luma
+	 * 0x8000 is intersitial. 0x10000 is sited with bottom luma
```

Comment:
If these are the expected values, then why is the range -1<<16, 1<<16 when creating the property with ```drm_property_create_signed_range```?

Defining as a signed range, and then putting apparently positive hex values in here is confusing.